client/src/Booking.jsx

```diff
@@ -99,30 +91,30 @@ class Booking extends React.Component {
 
   render() {
     return (
-      <div className="booking-widget hipbook" id="booking-widget">
-        <div className="booking-widget__banner">
-          <div className="wrapper">
-            <div className=".booking-widget__standard-price-wrapper">
-              <div>
-                <h5 className="booking-widget__price">${this.state.pricePerNight}</h5>
-                <span>per night (2 guests)</span>
+      <div className="widget-container">
+        <div className="booking-widget hipbook" id="booking-widget">
+          <div className="booking-widget__banner">
+            <div className="wrapper">
+              <div className=".booking-widget__standard-price-wrapper">
+                <div>
+                  <h5 className="booking-widget__price">${this.state.pricePerNight}</h5>
```

Comment:
the price per night disappeard on this PR. I think you can just use `{this.state.pricePerNight}` csans the $